src/ensembl/src/shared/eye-toggle-icon/EyeToggleIcon.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import eyeOnIcon from './eye-on.svg';
+import eyeOffIcon from './eye-off.svg';
+
+import styles from './EyeToggleIcon.scss';
+
+type Props = {
+  iconStatus: string;
+  description: string;
+  callbackFunction: () => void;
```

Comment:
`onClick` instead of `callbackFunction `?

Reply:
Or maybe `clickHandler` to differentiate between the prop and the React `onClick` event?

Reply:
There was a `handleClick` function in the component's implementation, which makes `clickHandler` potentially as confusing as `onClick` :-) I am happy to go with either, but think `onClick` is a slightly more natural name.

Reply:
To me it's more like asking "which external function needs to be called when this button is clicked?" and also it would differentiate it from the React onClick like @imran-ebi mentioned.

How about using `onClickHandler`?

Reply:
I would say, too verbose.